library/std/src/net/udp.rs

```diff
@@ -118,6 +138,164 @@ impl UdpSocket {
         self.0.recv_from(buf)
     }
 
+    /// Moves the socket to receive the TTL as control message in [`SocketAncillary`].
```

Comment:
"Moves" isn't the correct verb here. Also, for consistency with other functions like `set_ttl` and `set_broadcast`, the first paragraph (which serves as the one-line summary) should mention the option name and look like ```Sets the value of the `IP_RECVTTL` option for this socket.``` , and then the second paragraph should describe the effect of the option:

```
If enabled, received packets will come with ancillary data ([`SocketAncillary`]) providing the time-to-live (TTL) value of the packet.